.gitignore

```diff
@@ -23,6 +23,5 @@ nuget.exe
 *.ncrunchsolution
 *.*sdf
 *.ipch
-/SignalR-Server.sln.ide
-
-*.sln.ide
\ No newline at end of file
+*.sln.ide
+debugSettings.json
```

Comment:
debugSettings.json is a new one that is being recommended to add to the ignore file